### PR TITLE
Fix sort bug due to syntax error.

### PR DESCRIPTION
**Pull request by jkmarx**
fix-sort

### CODE REVIEW

File: refinery/data_set_manager/utils.py [hunk 1]
```diff
@@ -711,6 +711,9 @@ def generate_solr_params(
             facet_fields_obj[facet]['excludeTags'] = facet.upper()
         filter_arr.extend(create_facet_filter_query(facet_filter))
 
+    if sort:
+        fixed_solr_params['sort'] = sort
```
Comment (mccalluc) [inline refinery/data_set_manager/utils.py]: Maybe just add another line when you define the dict up top? Hopefully nothing depends on the absence of the key in `fixed_solr_params`? So get rid of this `if` and instead:
```python
     fixed_solr_params = {
         "facet.limit": "-1",
         "fq": "is_annotation:{}".format(is_annotation),
         "rows": row,
         "start": start,
         "sort": params.get('sort'),
         "wt": "json"
}
```

Or maybe not: I might be missing something

Comment (jkmarx) [reply refinery/data_set_manager/utils.py]: @mccalluc I'd have to check again, but pretty sure there was weirdness if I included sort: None. Let's leave it for now, and I'll recheck later.

Comment (jkmarx) [reply refinery/data_set_manager/utils.py]: Note, this is the not fully supported Solr JSON REQUEST API in our version

Comment (mccalluc) [reply refinery/data_set_manager/utils.py]: Go with what you know works: Thanks!

**Pull request merged**